docs/pages/admin-guides/teleport-policy/integrations/crown-jewels.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: See permission changes with Teleport Policy Crown Jewels
+description: Describes how to use Teleport Policy Crown Jewels to see permission changes in Teleport.
+---
+
+With Access Graph's Crown Jewel, you can see changes in user and resource permissions.
```

Comment:
This still feels off to me. The title of the doc refers to _Teleport Policy Crown Jewels_ (it doesn't mention access graph at all).

How about this for an intro paragraph?

>>> Teleport Policy's Crown Jewel feature allows you to track changes to access for your most sensitive users or resources. When you mark a resource as a Crown Jewel, Teleport emits audit events any time access to that resource changes. These audit events include snapshots of the permissions before and after the change, which can alert you of unexpected access changes and allow you to verify the results.

Reply:
> The title of the doc refers to Teleport Policy Crown Jewels (it doesn't mention access graph at all).

We removed "access graph" from most places in our docs already. TBH I'm not sure why we did that, but I also try to refer to all features as "Teleport Policy" not "Access Graph".
I'll try to figure out why we did that. If we want to change the naming, we should probably also change the name in the Teleport UI to keep it consistent.

Reply:
>How about this for an intro paragraph?

LGTM. I updated the docs.